tools/capput/capput.c

```diff
@@ -301,11 +301,11 @@ txput(void *arg)
         }
 
         if (killthreads || added == opts.chunk) {
-            int rc;
+            merr_t rc;
 
             ti->state = 'w';
             added = 0;
-            rc = pthread_barrier_wait(&put_barrier1);
+            rc = merr(pthread_barrier_wait(&put_barrier1));
             if (rc > 0)
```

Comment:
There was nothing wrong with this code:   `rc` was an `int` and `pthread_barrier_wait` returns an `int`.

Reply:
The problem is calling `fatal()` with an int.  So fix that with `fatal(merr(rc), ....)` or make a version of fatal that takes an `int`.